curvefs/src/metaserver/inode_manager.cpp

```diff
@@ -446,6 +441,13 @@ MetaStatusCode InodeManager::UpdateInode(const UpdateInodeRequest& request,
             return MetaStatusCode::STORAGE_INTERNAL_ERROR;
         }
     }
+
+    if (needAddTrash) {
+        inodeStorage_->UpdateDeletingKey(old, logIndex);
```

Comment:
add and delete op all deal in trash is better.

Reply:
fix